docs/developers/developing_iree/benchmarking.md

```diff
@@ -20,6 +20,7 @@ To use `iree-benchmark-module`, generate an IREE module for the target backend:
 
 ```shell
 $ bazel run //iree/tools:iree-translate -- \
+  -iree-input-type=mhlo \
   -iree-mlir-to-vm-bytecode-module \
   -iree-hal-target-backends=vmvx \
   $PWD/iree/tools/test/iree-benchmark-module.mlir \
```

Comment:
You could also update [`iree-benchmark-module.mlir`](https://github.com/google/iree/blob/main/iree/tools/test/iree-benchmark-module.mlir) and [`iree-run-mlir.mlir`](https://github.com/google/iree/blob/main/iree/tools/test/iree-run-mlir.mlir) (they are copies of `iree-run-module.mlir` but test different `// RUN:` commands). Then this example could also drop the `-iree-input-type=mhlo`.